src/core/dev_api/openvino/op/util/node_util.hpp

```diff
@@ -4,18 +4,29 @@
 
 #pragma once
 
+#include <string>
+
 #include "openvino/core/node.hpp"
 
 namespace ov {
-namespace op {
 namespace util {
+/**
+ * @brief Creates default tensor name for given Node's output.
+ * The name format is "node_name:output_port".
+ *
+ * @param output - Node's output to create name for tensor.
+ * @return Default tensor name.
+ */
+OPENVINO_API std::string default_tensor_name(const Output<const Node>& output);
```

Comment:
default_tensor_name sounds like variable
change to get_default_tensor_name

Reply:
Changed.